platform/jobs/edxEndToEndTestsJob.groovy

```diff
@@ -82,6 +82,20 @@ Map pipelineJob = [ name: 'edx-e2e-tests',
                     junitReportPath: 'reports/*.xml'
                     ]
 
+// The microsites-staging-tests job is run automatically on every deployment of the edx-platform
+// from the gocd pipeline.
+Map micrositespipelineJob = [ name: 'microsites-staging-tests',
```

Comment:
nit: I would capitalize the first "P" in "pipeline" to match the casing of the other items.